src/language-js/print/object.js

```diff
@@ -96,6 +147,7 @@ function printObject(path, options, print) {
     (node.type !== "ObjectPattern" &&
       options.objectWrap === "preserve" &&
       propsAndLoc.length > 0 &&
+      // FIXME
       hasNewlineInRange(
         options.originalText,
         locStart(node),
```

Comment:
@sosukesuzuki Any idea how to fix this? In other nodes they all starts with `{`, how do we find `{` here?